views/purchase.hbs

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <base href="/">
+    <!--link stylesheet-->
+    <link rel="stylesheet" href="/stylesheets/reset.css">
+    <link href="https://fonts.googleapis.com/css?family=Montserrat|Open+Sans" rel="stylesheet">
+    <link rel="stylesheet" href="/stylesheets/style.css">
+    <link rel="stylesheet" href="/stylesheets/stylePurchase.css">
+    <!--link script-->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    <title>Amazon Purchase</title>
+</head>
+
```

Comment:
You just need to put code inside <body> tag only.  Like said, rendering takes only <body> and place it into layout.hbs.    You can see {{{body}}} inside layout.hbs which means it takes <body> and place it there...